### PR TITLE
[C-2311] Fix notification timestamp logic

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Fixes notification timestamp logic and provides consistent experince between legacy and new system.
- Updates legacy timestamps to use unix timestamps, which is what discovery uses. This makes it easier to create appropriate request params, and sort notifs.
- Updates discovery notifications to use unix seconds, not milliseconds, which allows endpoint recent notifs.
- Adds query-string to make managing parameters much easier

### CODE REVIEW

File: packages/common/src/services/audius-backend/AudiusBackend.ts [hunk 1]
```diff
@@ -2103,12 +2105,22 @@ export const audiusBackend = ({
...
     }
   }
 
+  function mapIdentityNotification(
+    notification: IdentityNotification
+  ): Notification {
+    const { timestamp, ...restNotification } = notification
+    return {
+      ...restNotification,
+      timestamp: Date.parse(timestamp) / 1000
```
Comment (sabrina-kiam) [inline packages/common/src/services/audius-backend/AudiusBackend.ts:2120]: may be worth leaving a comment here explaining this conversion??

File: packages/common/src/store/notifications/notificationsSlice.ts [hunk 2]
```diff
@@ -11,6 +11,25 @@ import {
   UpdateNotificationsAction
 } from './types'
 
+// export const notificationsAdapter = createEntityAdapter<Notification>({
```
Comment (sabrina-kiam) [inline packages/common/src/store/notifications/notificationsSlice.ts:14]: do we need this??

File: packages/web/src/common/store/notifications/fetchNotifications.ts [hunk 3]
```diff
@@ -5,20 +5,23 @@ import {
...
+  // unix timestamp
+  timeOffset?: number
   groupIdOffset?: string
 }
 
 export function* fetchNotifications(config: FetchNotificationsParams) {
-  const { limit, timeOffset, groupIdOffset } = config
+  const {
+    limit,
+    timeOffset = Math.round(new Date().getTime() / 1000),
```
Comment (sabrina-kiam) [inline packages/web/src/common/store/notifications/fetchNotifications.ts:22]: same here - may be worth putting a comment why we do this conversion

File: packages/web/src/common/store/notifications/fetchNotifications.ts [hunk 4]
```diff
@@ -53,11 +56,9 @@ export function* fetchNotifications(config: FetchNotificationsParams) {
   const shouldFetchNotificationFromDiscovery =
     useDiscoveryNotifications &&
     discoveryNotificationsGenesisTimestamp &&
-    Date.parse(timeOffset) > discoveryNotificationsGenesisTimestamp
+    timeOffset > discoveryNotificationsGenesisTimestamp
```
Comment (sabrina-kiam) [inline packages/web/src/common/store/notifications/fetchNotifications.ts:59]: so `discoveryNotificationsGenesisTimestamp` is also a unix timestamp??

File: packages/common/src/services/audius-backend/AudiusBackend.ts [hunk 5]
```diff
@@ -2448,27 +2465,35 @@ export const audiusBackend = ({
...
+        limit,
+        handle,
+        withSupporterDethroned: withDethroned,
+        withTips: true,
+        withRewards: true,
+        withRemix: true,
+        withTrendingTrack: true
+      }
+
+      const getNotificationsUrl = queryString.stringifyUrl({
```
Comment (sabrina-kiam) [inline packages/common/src/services/audius-backend/AudiusBackend.ts:2484]: this is awesome

**Review by sabrina-kiam — APPROVED** [review]
great fix!

File: packages/common/src/services/audius-backend/AudiusBackend.ts (hunk 1 above)
Comment (dylanjeffers) [reply packages/common/src/services/audius-backend/AudiusBackend.ts:2120]: totally!

File: packages/common/src/store/notifications/notificationsSlice.ts (hunk 2 above)
Comment (dylanjeffers) [reply packages/common/src/store/notifications/notificationsSlice.ts:14]: no, thanks :)

File: packages/web/src/common/store/notifications/fetchNotifications.ts (hunk 3 above)
Comment (dylanjeffers) [reply packages/web/src/common/store/notifications/fetchNotifications.ts:22]: for sure :)

File: packages/common/src/services/audius-backend/AudiusBackend.ts (hunk 5 above)
Comment (dylanjeffers) [reply packages/common/src/services/audius-backend/AudiusBackend.ts:2484]: yesss so much cleaner right?

**Pull request merged**